evaluation/log_experiments.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+import comet_ml.integration.pytorch
+from comet_ml import Experiment
+import torch
+import sklearn
+from models.model import BaseModel
+from models.sklearn_model import SklearnModel
+from models.trend_model import TrendModel
+from models.naive_models import AverageYieldModel
+from models.nn_models import BaseNNModel
+
+# get paths
+evaluation_path = os.path.dirname(os.path.realpath(__file__))
+root_path = evaluation_path[:-10]
```

Comment:
Could you add a note why -10?

Reply:
Sure!
Basically it's to remove the word "evaluation" to get the root_path.
A bit hacky maybe, but we can refactor later if needed.